.stylelintrc.json

```diff
@@ -41,7 +41,7 @@
     "font-family-name-quotes": "always-where-recommended",
     "font-family-no-duplicate-names": true,
     "font-weight-notation": "numeric",
-    "function-blacklist": "rgb",
+    "function-disallowed-list": "rgb",
```

Comment:
It looks like we have to:

1. Pin the version of Stylelint in peerdeps.

2. Bump the version of the config to the next major one.

So let's do it.

Also add a note to the READMEs about the Stylelint as a peer dep.

Reply:
Thanks. I've fixed that.
Also I have updated all dependencies of this package.